common/lib/dependabot/file_fetchers/base.rb

```diff
@@ -26,8 +26,11 @@ class Base
         Dependabot::Clients::CodeCommit::NotFound
       ].freeze
 
-      GIT_SUBMODULE_ERROR_REGEX =
+      GIT_SUBMODULE_ERROR_REGEX1 =
         /^fatal: unable to access '(?<url>.*)': The requested URL returned error: (?<code>\d+)$/
+      GIT_SUBMODULE_ERROR_REGEX2 =
+        /^fatal: clone of '(?<url>.*)' into submodule path '.*' failed$/
+      GIT_SUBMODULE_ERROR_REGEX = /(#{GIT_SUBMODULE_ERROR_REGEX1})|(#{GIT_SUBMODULE_ERROR_REGEX2})/
```

Comment:
I like you're approach here. Could we use constant names that reflect the underlying errors more directly? Maybe something like `GIT_SUBMODULE_INACCESSIBLE_ERROR` for `REGEX1` and `GIT_SUBMODULE_CLONE_ERROR` for `REGEX2`?